src/AEPi/codecs/EtcPakCodec.py

```diff
@@ -12,38 +14,81 @@
+)
 class EtcPakCodec(ImageCodecAdaptor):
     @classmethod
-    def compress(cls, im: Image, format: CompressionFormat, quality: Optional[CompressionQuality]) -> bytes:
+    def compress(
+        cls, im: Image, format: CompressionFormat, quality: Optional[CompressionQuality]
+    ) -> bytes:
         imageIn, ctx = cls._ensureRgba(im)
         with ctx:
             if format is CompressionFormat.DXT5:
-                return etcpak.compress_to_dxt5(imageIn.tobytes(), imageIn.width, imageIn.height) # type: ignore[reportUnknownVariableType]
+                return etcpak.compress_to_dxt5(imageIn.tobytes(), imageIn.width, imageIn.height)  # type: ignore[reportUnknownVariableType]
 
             elif format is CompressionFormat.ETC1:
-                return etcpak.compress_to_etc1(imageIn.tobytes(), imageIn.width, imageIn.height) # type: ignore[reportUnknownVariableType]
-            
+                return etcpak.compress_to_etc1(imageIn.tobytes(), imageIn.width, imageIn.height)  # type: ignore[reportUnknownVariableType]
+
             elif format is CompressionFormat.ETC2:
                 # etcpak does provide a function for etc2 compression, but it produces almost completely black images
                 # ETC2 is backwards compatible with ETC1, so as a stopgap we'll just compress as etc1
                 # https://www.khronos.org/assets/uplo...pengl-es-bof/Ericsson-ETC2-SIGGRAPH_Aug12.pdf
-                return etcpak.compress_to_etc1(imageIn.tobytes(), imageIn.width, imageIn.height) # type: ignore[reportUnknownVariableType]
-            
-        raise ValueError(f"Codec {EtcPakCodec.__name__} does not support format {format.name}")
-    
-    
+                return etcpak.compress_to_etc1(imageIn.tobytes(), imageIn.width, imageIn.height)  # type: ignore[reportUnknownVariableType]
+
+        raise ValueError(
+            f"Codec {EtcPakCodec.__name__} does not support format {format.name}"
+        )
+
+    @classmethod
+    def decompress(
+        cls,
+        fp: bytes,
+        format: CompressionFormat,
+        width: int,
+        height: int,
+        quality: Optional[CompressionQuality],
+    ) -> Image:
+        if format not in ETCPAK_FORMAT_MAP:
+            raise ValueError(
+                f"Codec {EtcPakCodec.__name__} does not support format {format.name}"
+            )
+        match format:
+            case CompressionFormat.DXT1:
+                decompressed = etcpak.decompress_bc1(fp, width, height)
+            case CompressionFormat.DXT5:
+
+                decompressed = etcpak.decompress_bc3(fp, width, height)
+            case CompressionFormat.ETC1:
+
+                decompressed = etcpak.decompress_etc1_rgb(fp, width, height)
+            case CompressionFormat.ETC2:
+
+                decompressed = etcpak.decompress_etc2_rgb(fp, width, height)
+        im = PIL.Image.frombytes("RGBA", (width, height), decompressed, "raw")  # type: ignore[reportUnknownMemberType]
```

Comment:
```suggestion
        im = Image.frombytes("RGBA", (width, height), decompressed, "raw") # type: ignore[reportUnknownMemberType]
```

Small error found by the linter.